etc/deploy/cloudfront/secure-cloudfront.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+CLOUDFRONT_OAI_ID=
+set -euxo pipefail
+
+parse_flags() {
+  # Check prereqs
+  which aws
+  which jq
+  which uuid
+  # Common config
+  export AWS_REGION=us-east-1
+  export AWS_AVAILABILITY_ZONE=us-east-1a
+
+  # Parse flags
+  eval "set -- $( getopt -l "state:,region:,zone:,bucket:,distribution:,cloudfront-keypair-id:,cloudfront-private-key-file:" "--" "${0}" "${@}" )"
+  while true; do
+      case "${1}" in
+          --region)
+            export AWS_REGION="${2}"
+            shift 2
+            ;;
+          --zone)
+            export AWS_AVAILABILITY_ZONE="${2}"
+            shift 2
+            ;;
+          --bucket)
+            export BUCKET="${2}"
+            shift 2
+            ;;
+          --distribution)
+            export DISTRIBUTION="${2}"
+            shift 2
+            ;;
+          --cloudfront-keypair-id)
+            export CLOUDFRONT_KEYPAIR_ID="${2}"
+            shift 2
+            ;;
+          --cloudfront-private-key-file)
+            export CLOUDFRONT_PRIVATE_KEY_FILE="${2}"
+            shift 2
+            ;;
+          --)
+            shift
+            break
+            ;;
+      esac
+  done
+
+  if [ $CLOUDFRONT_KEYPAIR_ID -n ]; then
```

Comment:
Because of the incantation, this line will err instead of continuing. I'll just add some print statements that will serve as emptiness checks